src/warnet/admin.py

```diff
@@ -33,3 +37,93 @@ def init():
         f"[green]Copied network and namespace example files to {Path(current_dir) / NETWORK_DIR.name}[/green]"
     )
     richprint(f"[green]Created warnet project structure in {current_dir}[/green]")
+
+
+@admin.command()
+@click.option(
+    "--kubeconfig-dir",
+    default="kubeconfigs",
+    help="Directory to store kubeconfig files (default: kubeconfigs)",
+)
+@click.option(
+    "--token-duration",
+    default=172800,
+    type=int,
+    help="Duration of the token in seconds (default: 48 hours)",
+)
+def create_kubeconfigs(kubeconfig_dir, token_duration):
+    """Create kubeconfig files for all ServiceAccounts in warnet team namespaces starting with <prefix>."""
+    kubeconfig_dir = os.path.expanduser(kubeconfig_dir)
+
+    cluster_name = get_kubeconfig_value("{.clusters[0].name}")
+    cluster_server = get_kubeconfig_value("{.clusters[0].cluster.server}")
+    cluster_ca = get_kubeconfig_value("{.clusters[0].cluster.certificate-authority-data}")
+
+    os.makedirs(kubeconfig_dir, exist_ok=True)
+
+    # Get all namespaces that start with prefix
+    # This assumes when deploying multiple namespacs for the purpose of team games, all namespaces start with a prefix,
+    # e.g., tabconf-wargames-*. Currently, this is a bit brittle, but we can improve on this in the future
```

Comment:
I'm still pretty worried about the tech debt from this mechanism since it is so ubiquitous in this PR. Hard-coding the k8s namespaces and logging namesapce as well... and forcing the user to use a naming convention... all seems to go against our design principles!

Reply:
Agree that the prefixes is an anti pattern. I was looking at using namespace labels so they we could do things like `get_namespaces_by_type(WARNET_TANKS)` / `get_namespaces_by_type(LOGGING)` / `get_namespaces_by_type(ALL)`, etc but that felt like it would take more work to get right. If the tradeoff is we have prefixed namespaces and can use namespaces for Tabconf, vs we don't have namespaces for Tabconf at all, I'm more inclined to proceed as is and have a backlog to refactor to using labels (happy to own following up on that).